cli/src/stake.rs

```diff
@@ -34,13 +34,13 @@ use std::{ops::Deref, sync::Arc};
 pub const STAKE_AUTHORITY_ARG: ArgConstant<'static> = ArgConstant {
     name: "stake_authority",
     long: "stake-authority",
-    help: "Public key of authorized staker (defaults to cli config pubkey)",
+    help: "Authorized staker (defaults to cli config keypair)",
```

Comment:
nit: make our manual default look like what clap generates
```suggestion
    help: "Authorized staker [default: cli config keypair]",
```

Reply:
I bet there are a few other uses of that format. I'll update all.